anvil/sample.py

```diff
@@ -159,6 +159,9 @@ def chain_autocorrelation(
 
     # Sample some states
     _, history = sample_batch(loaded_model, action, batch_size, thermalised_state)
+    
+    accepted = float(torch.sum(history))
+    #print(f"Acceptance: {accepted / batch_size}")
```

Comment:
do we need a commented out print statement?

Reply:
Not really. It's there because my current hacky script which runs anvil-train, anvil-sample x 10 until a given acceptance uses these print statements. I'm sure there's a better way to do this but I just needed to get something working.

I'll take them out of this branch though!